content/e/ptw/2024-10-19-ptw-underground-23.md

```diff
@@ -22,6 +22,10 @@ city = "Kozłów"
   - '???'
   - s: Trick or Treat Hardcore Match
     nc: Upcoming
+- - '[Alex Brave](@/w/alex-brave.md)'
+  -  g: true
+  -  s: The Brave Show - episode 1
+  -  nc: Upcoming    
```

Comment:
In a block-style map that is an element of the list, only the first item needs to be prefixed with a dash.

```suggestion
  -  g: true
     s: The Brave Show - episode 1
     nc: Upcoming    
```